homeassistant/components/cloud/__init__.py

```diff
@@ -373,17 +395,24 @@ async def _on_initialized() -> None:
 
 
 @callback
-def _remote_handle_prefs_updated(cloud: Cloud[CloudClient]) -> None:
-    """Handle remote preferences updated."""
+def _handle_prefs_updated(hass: HomeAssistant, cloud: Cloud[CloudClient]) -> None:
+    """Handle preferences updated."""
     cur_pref = cloud.client.prefs.remote_enabled
+    cur_cloudhooks = cloud.client.prefs.cloudhooks
     lock = asyncio.Lock()
 
     # Sync remote connection with prefs
     async def remote_prefs_updated(prefs: CloudPreferences) -> None:
         """Update remote status."""
         nonlocal cur_pref
+        nonlocal cur_cloudhooks
 
         async with lock:
+            if cur_cloudhooks != prefs.cloudhooks:
```

Comment:
Where is `cur_cloudhooks` updated?

Reply:
I've just fixed that sorry, you've review before my last commit. I was waiting for the CI to be green again.